lib/DDG/Spice/TwitchStreams.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::TwitchStreams;
+use strict;
+use DDG::Spice;
+
+name "TwitchStreams";
+source "http://www.twitch.tv";
+description "Get Twitch Stream info";
+primary_example_queries "twitch streams dota 2";
+category "special";
+topics "special_interest";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/TwitchStreams/TwitchStreams.pm";
+attribution github => ["https://github.com/dwaligon", "Jason O'Donnell"];
+
+triggers startend => ('twitch streams',
+		              'streams twitch',
+		              'twitchtv streams');
```

Comment:
@Dwaligon it's alright if `twitch.tv` triggers the TVMaze IA -- triggers are allowed to overlap. For the purpose of relevancy though, we should add a check to TVMaze so that it doesn't return if there's no remainder.